Generators/Generators.gradle

```diff
@@ -215,8 +215,8 @@ def pythonStaticMethodsArgs = ['io.deephaven.util.calendar.Calendars',
                                'io.deephaven.db.tables.utils.DBTimeUtils',
                                     '/Generators/src/main/java/io/deephaven/pythonPreambles/DBTimeUtilsPreamble.txt',
                                     '/Integrations/python/deephaven/DBTimeUtils/__init__.py',
-                               'io.deephaven.db.tables.select.QueryScope',
-                                    '/Integrations/python/deephaven/QueryScope/__init__.py',
+//                               'io.deephaven.db.tables.select.QueryScope',
+//                                    '/Integrations/python/deephaven/QueryScope/__init__.py',
```

Comment:
Just delete it if we aren't using it.

Reply:
Confirm that there aren't any header files for this case that need deleting.  I don't think there are, but just making sure.

Reply:
deleted and global search all didn't find any leftover references to QueryScope.